reporter/internal/pdata/generate.go

```diff
@@ -36,15 +37,16 @@ type uniqueMapping struct {
 // Generate generates a pdata request out of internal profiles data, to be
 // exported.
 func (p *Pdata) Generate(tree samples.TraceEventsTree,
-	agentName, agentVersion string) (pprofile.Profiles, error) {
+	agentName, agentVersion string,
+) (pprofile.Profiles, error) {
 	profiles := pprofile.NewProfiles()
 	dic := profiles.ProfilesDictionary()
 
 	// Temporary helpers that will build the various tables in ProfilesDictionary.
-	stringSet := make(OrderedSet[string], 64)
-	funcSet := make(OrderedSet[funcInfo], 64)
-	mappingSet := make(OrderedSet[uniqueMapping], 64)
-	locationSet := make(OrderedSet[locationInfo], 64)
+	stringSet := make(orderedset.OrderedSet[string], 64)
+	funcSet := make(orderedset.OrderedSet[funcInfo], 64)
+	mappingSet := make(orderedset.OrderedSet[uniqueMapping], 64)
+	locationSet := make(orderedset.OrderedSet[locationInfo], 64)
```

Comment:
We need to add an empty first element to `dic.FunctionTable`, `dic.AttributeTable` and `dic.LocationTable`